### PR TITLE
Genome uploader

### PR DESCRIPTION
**Pull request by Ge94**
Content-wise everything should be there, could you please double check whether it looks confusing?

Thank you!!

### CODE REVIEW

File: sessions/genome_uploader.qmd [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+Your task is to generate a tsv table with sample metadata describing the genomes you want to register and submit.
+:::
+
+Similarly to yesterday's practical, you will be free to insert values as you wish, as long as regular expressions and mandatory fields described in the checklist are respected. You will need to select whether you are uploading bins or MAGs, and select your checklist accordingly. 
+
+:::{.callout-note}
+To register a sample, a relative set of metadata must be filled according to the selected checklist. Some of them are mandatory, while some others are only recommended. The `genome_uploader` will automatically pick the right checklist depending on the input flag:
+
+* __GSC MIMAG__ for MAG samples (`-mag`)
+* __ENA binned metagenome__ for binned samples (`-bin`)
```
Comment (SandyRogers) [inline sessions/genome_uploader.qmd:43]: worth linking to the checklists on ena website?

Comment (SandyRogers) [reply sessions/genome_uploader.qmd:43]: never mind I see you do it below :D

File: sessions/genome_uploader.qmd [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+:::{.callout-tip}
+Taxonomies can be listed in either NCBI or GTDB format. An example of valid taxonomies you could use in this scenario could be:
+
+* GTDB: `d__Bacteria;p__Proteobacteria;c__Gammaproteobacteria;o__Enterobacterales;f__Vibrionaceae;g__Photobacterium;s__Photobacterium piscicola`
+* NCBI: `1;131567;2759;33154;4751;451864;5204;452284;1538075;162474;742845;55193;76775`
+:::
+
+## Provided materials
+You can follow [this link](https://docs.google.com/spreadsheets/d/1eT05boK8MkGq6MNOBMa1w0qHD-n5e5aU03_nu4CBBiA/edit?usp=sharing) to find a Google Spreasheet called `genome_uploader_template`.
```
Comment (SandyRogers) [inline sessions/genome_uploader.qmd:104]: brilliant

**Review by SandyRogers — APPROVED** [review]
Excellent, thank you @Ge94 !

File: sessions/genome_uploader.qmd [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+cd /home/training/genomeuploader/
+```
+
+## The genome_uploader and its metadata handling
+
+As Maira thoroughly explained in the theory session, bin and MAG submission undergoes 3 different steps:
+
+* Register binned and/or MAG samples
+* Generate manifest files
+* Submit assemblies with Webin-CLI
```
[the remark this answers is not in the document]
Comment (Ge94) [reply sessions/genome_uploader.qmd:29]: I literally copy pasted the content from Maira's slide - it's always been very confusing for me too, but I better use their terminology

**Pull request merged**